app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -0,0 +1,81 @@
+package org.jellyfin.androidtv.util.profile
+
+import org.jellyfin.androidtv.constant.CodecTypes
+import org.jellyfin.androidtv.constant.ContainerTypes
+import org.jellyfin.androidtv.util.Utils
+import org.jellyfin.apiclient.model.dlna.CodecProfile
+import org.jellyfin.apiclient.model.dlna.CodecType
+import org.jellyfin.apiclient.model.dlna.DeviceProfile
+import org.jellyfin.apiclient.model.dlna.ProfileCondition
+import org.jellyfin.apiclient.model.dlna.ProfileConditionType
+import org.jellyfin.apiclient.model.dlna.ProfileConditionValue
+import org.jellyfin.apiclient.model.dlna.SubtitleDeliveryMethod
+import org.jellyfin.apiclient.model.dlna.SubtitleProfile
+import org.jellyfin.apiclient.model.dlna.TranscodingProfile
+import timber.log.Timber
+
+object ProfileHelper {
+	private val MediaTest = MediaCodecCapabilitiesTest()
```

Comment:
```suggestion
	private val MediaTest by lazy { MediaCodecCapabilitiesTest() }
```

So it is initiated on first use